guides/source/action_text_overview.md

```diff
@@ -97,6 +97,8 @@ Then refer to this field in the form for the model:
 <% end %>
 ```
 
+NOTE: The `rich_text_area` form helper automatically adds "trix-content" as a class to the trix-editor element. This class ensures [default styling is applied](https://github.com/rails/rails/blob/adc5eb66f861a98da32f7c682155360d5b9a4a52/actiontext/app/helpers/action_text/tag_helper.rb#L14). Adding your own classes to the form helper will remove "trix-content" as a class.
```

Comment:
```suggestion
**Note:** The `rich_text_area` form helper automatically adds the `trix-content` CSS class to the Trix editor element. `trix-content` is responsible for the editor's [default styling](https://github.com/rails/rails/blob/adc5eb66f861a98da32f7c682155360d5b9a4a52/actiontext/app/helpers/action_text/tag_helper.rb#L14). Passing a `class` argument overrides `trix-content`, removing the default styles.
```

We should be consistent with the way `**Note:**` was formatted above. I also added a few suggestions that I think improve clarity.